CHANGELOG.md

```diff
@@ -7,19 +7,6 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
-## [0.15.3] - 2022-11-04
```

Comment:
These lines should not be deleted.

I think this is caused by the following steps.
```
git checkout topolvm.io
git diff main > /tmp/topolvm.io.patch
```

I guess the base of  `topolvm.io` branch is too old. You had to rebase it first as follows.
```
git checkout main
git pull
git checkout topolvm.io
git rebase main
git diff main > /tmp/topolvm.io.patch
```

Reply:
Sorry, I'll fix it.

Reply:
Thanks.
We need to move to the new PR https://github.com/topolvm/topolvm/pull/592 ?
It looks good.
After it become ready for review, I will review it.

Reply:
> We need to move to the new PR https://github.com/topolvm/topolvm/pull/592 ?

Yes, please.